docker/tests/unit/services/test_search_processor.py

```
@@ -0,0 +1,91 @@
+        api_result = APISearchResult(
+            title="Original", link="https://orig.com", snippet="Original snippet"
+        )
+        mock_scrape.return_value = SearchResult(
+            title="Original",
+            url="https://orig.com",
+            snippet="Original snippet",
+            content="Scraped",
+        )
+
+        # Act
+        results = process_search_results([api_result])
```

Ruff (0.13.2)

Local variable `results` is assigned to but never used

Remove assignment to unused variable `results`

(F841)